### PR TITLE
Fix kubemark issue & Add document for kubemark test

### PR DESCRIPTION
**Pull request by TommyLike**
**What this PR does / why we need it**:
1. Fix kubemark issue when testing on GCE
2. Add document for it.

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by TommyLike** [comment]
@k82cn  @Rajadeepan  please take a look.

**Comment by k82cn** [comment]
@Rajadeepan @asifdxtreme , please take a look; we need have benchmark result for each release.

File: test/kubemark/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Guide for running kubemark with kube-batch scheduler
+
+## 1.Pre-requisites
+There are several requirements before installing kubemark cluster for test.
+1. **Docker**: Building kubernetes releases with command``make quick-release`` require having a docker environment.
+2. **Google cloud SDKs**: Kubemark utilize GCE to create its master node in default, therefore you need ensure
+   google cloud SDKs are installed on your environment. **NOTES**: ``gcloud beta`` is required in addition.
+3. **External Kubernetes cluster**: Kubemark utilize external kubernetes cluster to hold its ``Hollo Nodes``, before
```
Comment (Rajadeepan) [inline test/kubemark/README.md]: nit: s/Hollo/Hollow

**Comment by Rajadeepan** [comment]
@TommyLike  Looks fine.

**Comment by TommyLike** [comment]
@k82cn  @Rajadeepan need lgtm label.

**Pull request merged**